x-pack/plugins/apm/server/lib/alerts/register_transaction_duration_alert_type.ts

```diff
@@ -92,8 +94,16 @@ export function registerTransactionDurationAlertType({
         savedObjectsClient: services.savedObjectsClient,
       });
 
+      const searchAggregatedTransactions =
+        config['xpack.apm.searchAggregatedTransactions'] !==
+        SearchAggregatedTransactionSetting.never;
+
+      const index = searchAggregatedTransactions
+        ? indices['apm_oss.metricsIndices']
+        : indices['apm_oss.transactionIndices'];
```

Comment:
Would it make sense to extract this to a function so it can be re-used between the alerts?
And add a comment to make the behaviour around `auto` clear.

Reply:
Seems overkill for two usages? I'll add a comment anyway.